test/swoosh/test_assertions_test.exs

```diff
@@ -123,25 +92,22 @@ defmodule Swoosh.TestAssertionsTest do
 
   test "assert email not sent with unexpected email" do
     unexpected_email = new() |> subject("Testing Avenger")
+
     assert_email_not_sent(unexpected_email)
   end
 
   test "assert email not sent with expected email", %{email: email} do
-    message =
-      "Unexpectedly received message {:email, #{inspect(email)}} (which matched {:email, ^email})"
+    message = ~r/Expected[\s\S]+to not contain[\s\S]+but this email matched/
```

Comment:
`[\s\S]+` is just `.+`? 😄

Reply:
`.` doesn't match newlines we have in messages, `[\s\S]` is the next best thing.

Reply:
Thanks for the clarification. Have you tried `~r/Expected.+to not contain/m`? Note the `m` at the end, the `multiline` modifier.

Reply:
Actually my bad. Should be `s`.